src/database/dto/BookingDataDTO.java

```diff
@@ -61,9 +80,14 @@ public BigDecimal getPrice() {
 		return price;
 	}
 	
-	@Override
+	/*@Override
 	public String toString() {
 		String formattedDateTime=entryDate.toString();
 		return String.format("id: %s and date: %s", this.id+formattedDateTime);
+	}*/
+	
+	@Override
+	public String toString() {
+		return "BookingDataDTO"+"Id="+this.id+"entrydate="+this.entryDate +"departuredate="+this.departureDate+"price="+this.price+"methodp="+this.paymentMethod;
```

Comment:
`"methodp="+this.paymentMethod` should be `"paymentMethod="+this.paymentMethod`.